fe/fe-core/src/main/java/com/starrocks/journal/JournalEntity.java

```diff
@@ -505,6 +510,26 @@ public void readFields(DataInput in) throws IOException {
                 isRead = true;
                 break;
             }
+            case OperationType.OP_CREATE_TASK:
+                data = Task.read(in);
+                isRead = true;
+                break;
+            case OperationType.OP_DROP_TASKS:
+                data = MultiDropTaskInfo.read(in);
+                isRead = true;
+                break;
+            case OperationType.OP_TASK_RUN_CREATE_STATUS:
+                data = TaskRunStatus.read(in);
+                isRead = true;
+                break;
+            case OperationType.OP_TASK_RUN_STATUS_CHANGE:
+                data = TaskRunStatusChange.read(in);
+                isRead = true;
+                break;
+            case OperationType.OP_DROP_TASK_RUNS:
+                data = MultiDropTaskRunInfo.read(in);
+                isRead = true;
+                break;
```

Comment:
we need a unified name, like
OP_CREATE_TASK_RUN
OP_ALTER_TASK_RUN
OP_DROP_TASK_RUNS